### PR TITLE
Add min deposit condittion; Fix decimal places for mock contract

### PR DESCRIPTION
**Pull request by ognjenkurtic**

### CODE REVIEW

File: contracts/BaseledgerUBTSplitter.sol [hunk 1]
```diff
@@ -96,7 +96,7 @@ contract BaseledgerUBTSplitter is Context, Ownable {
         public
         emptyString(baseledgerDestinationAddress)
     {
-        require(amount > 0, "amount should be greater than zero");
+        require(amount >= 100000000, "amount should be >= 1");
```
Comment (skosito) [inline contracts/BaseledgerUBTSplitter.sol:99]: this way it is not configurable, we discussed owner function to change it potentially? or will always be 1?

File: contracts/mocks/UBTMock.sol [hunk 2]
```diff
@@ -5,9 +5,9 @@ import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 contract UBTMock is ERC20, Ownable {
-    uint256 private constant SUPPLY = 1_000_000_000 * 10**18;
+    uint256 private constant SUPPLY = 1_000_000_000 * 10**8;
 
-    constructor() ERC20("LeagueDAO Governance Token", "LEAG") {
+    constructor() ERC20("UBT Mock Token", "LEAG") {
```
Comment (skosito) [inline contracts/mocks/UBTMock.sol:10]: nit, but also change LEAG to something, MUBT or similar

**Review by skosito — APPROVED** [review]
approved, but please check comments
also please run prettier before merging

File: contracts/BaseledgerUBTSplitter.sol (hunk 1 above)
Comment (ognjenkurtic) [reply contracts/BaseledgerUBTSplitter.sol:99]: ah,  forgot about this. will expand.

**Pull request merged**